Skype/SDN/articles/from-element-startorupdatetype-complextype-skype-for-business-sdn-interface-2-2-schema-c.md

```diff
@@ -1,16 +1,17 @@
 ﻿---
-title: From element (StartOrUpdateType complexType) 
+title: From element (StartOrUpdateType complexType) (Schema C)
 TOCTitle: From element (StartOrUpdateType complexType)
 ms:assetid: 75f762ae-fe25-fc78-7b61-57d4a6464f20
 ms:mtpsurl: https://msdn.microsoft.com/library/Mt404765(v=office.16)
 ms:contentKeyID: 68250711
+description: Source of the media stream.
```

Comment:
Descriptions should be 100-150 characters. Too short as is.

Reply:
Not enough info in article to create a 100 character description.